stdlib/3/urllib/parse.pyi

```diff
@@ -97,9 +97,16 @@ class SplitResultBytes(_SplitResultBytesBase, _NetlocResultMixinBytes): ...
 class ParseResultBytes(_ParseResultBytesBase, _NetlocResultMixinBytes): ...
 
 
+@overload
 def parse_qs(qs: str, keep_blank_values: bool = ..., strict_parsing: bool = ..., encoding: str = ..., errors: str = ...) -> Dict[str, List[str]]: ...
```

Comment:
Thanks! I didn't realize that `AnyStr` enforced that the type remained constant. The name seemed to imply that it was more permissive than that.

-